pwndbg/commands/heap.py

```diff
@@ -51,13 +51,14 @@ def heap(addr=None):
     """
     Prints out all chunks in the main_arena, or the arena specified by `addr`.
     """
+
     main_heap   = pwndbg.heap.current
     main_arena  = main_heap.get_arena(addr)
 
     if main_arena == None:
         return
 
-    heap_base = main_heap.get_bounds()[0]
+    heap_base = main_heap.get_bounds(addr)[0]
```

Comment:
Can `get_bounds` fail?  It may be worth adding an error message so we don't accidentally call `[0]` on `None` or something